api/src/pcapi/core/users/external/sendinblue.py

```diff
@@ -377,52 +355,8 @@ def add_contacts_to_list(user_emails: Iterable[str], sib_list_id: int, clear_lis
     configuration = sib_api_v3_sdk.Configuration()
     configuration.api_key["api-key"] = settings.SENDINBLUE_API_KEY
     contacts_api_instance: ContactsApi = sib_api_v3_sdk.ContactsApi(sib_api_v3_sdk.ApiClient(configuration))
-    process_api_instance: ProcessApi = sib_api_v3_sdk.ProcessApi(sib_api_v3_sdk.ApiClient(configuration))
-
-    if clear_list_first:
-        # https://developers.sendinblue.com/reference/removecontactfromlist
-        remove_contact = sib_api_v3_sdk.RemoveContactFromList(all=True)
-
-        try:
-            remove_response: PostContactInfo = contacts_api_instance.remove_contact_from_list(
-                sib_list_id, remove_contact
-            )
-            logger.info(
-                "ContactsApi->remove_contact_from_list returned: %s",
-                remove_response,
-                extra={"list_id": sib_list_id, "process_id": remove_response.contacts.process_id},
-            )
-
-            # While developing, this process takes up to 25 seconds to remove 2 contacts from the list!
-            # We must wait until process LIST_USERS_DELETE is completed, otherwise it may remove a contact which was
-            # re-added in next process IMPORTUSER
-            if remove_response.contacts.process_id:
-                _wait_for_process(process_api_instance, remove_response.contacts.process_id)
-
-        except SendinblueApiException as exception:
-            if exception.status == 524:
-                # Handled first because response is a generic html page, not the expected json body
-                logger.exception(
-                    "Timeout when calling ContactsApi->remove_contact_from_list",
-                    extra={"list_id": sib_list_id},
-                )
-                return False
-            try:
-                message = json.loads(exception.body).get("message")
-            except json.JSONDecodeError:
-                message = exception.body
-            if exception.status == 400 and message == "Contacts already removed from list and/or does not exist":
-                logger.info(
-                    "ContactsApi->remove_contact_from_list: list was already empty",
-                    extra={"list_id": sib_list_id},
-                )
-            else:
-                logger.exception(
-                    "Exception when calling ContactsApi->remove_contact_from_list: %s",
-                    exception,
-                    extra={"list_id": sib_list_id},
-                )
-                return False
+
+    iteration = 1
```

Comment:
C'est étonnant de commencer à 1 en informatique non ?
Tu pense que l'itération 0 n'aurait pas de sens ?

Reply:
C'est un compteur pour l'affichage dans les logs, et non un index de tableau

Reply:
Oui oui je vois bien l'usage
Mais on compte tout le temps à partir de 0 en informatique non ?
Bon ça n'a pas trop d'importance toute façon

Reply:
Hmmm on compte à partir de 0 lorsqu'il s'agit d'un index, qui représente un _offset_. Mais ce n'est pas le cas ici, il n'y a pas cette notion donc il me paraît naturel d'afficher à partir de 1 puisque c'est un nombre d'appels. Mais si tu souhaites commencer à zéro, je change...

Reply:
En fait si j'avais connaissance du nombre total d'appels au départ, j'aurais affiché dans les logs « 1 sur 2 », « 2 sur 2 ». Dans ce cas c'est évident qu'on ne mettrait pas « 0 sur 2 » et « 1 sur 2 ».